UnityProject/Assets/Scripts/Items/Weapons/Melee/MeleeEffect.cs

```diff
@@ -118,14 +118,22 @@ public void ServerPerformInteraction(HandApply interaction)
 			Vector2 dir = (target.transform.position - performer.transform.position).normalized;
 
 			WeaponNetworkActions wna = performer.GetComponent<WeaponNetworkActions>();
+			ToggleableEffect toggleableEffect = gameObject.GetComponent<ToggleableEffect>();
 
 			// If we're on harm intent we deal damage!
 			// Note: this has to be done before the teleport, otherwise the target may be moved out of range.
-			if (interaction.Intent == Intent.Harm)
+			if (interaction.Intent == Intent.Harm && toggleableEffect.CurrentWeaponState == ToggleableEffect.WeaponState.On)
 			{
 				// Direction of attack towards the attack target.
 				wna.ServerPerformMeleeAttack(target, dir, interaction.TargetBodyPart, LayerType.None);
 			}
+			else
+			{
+				Chat.AddActionMsgToChat(interaction.Performer,
+					$"You attempt to prod {interaction.TargetObject.ExpensiveName()} but the {gameObject.ExpensiveName()} was off!",
+					$"{interaction.Performer.ExpensiveName()} prods {interaction.TargetObject.ExpensiveName()}, luckily the {gameObject.ExpensiveName()} was off!");
```

Comment:
This will trigger when not Harm but On and then the message doesnt make sense.

Reply:
Ah good point, will fix it after I'm done looking at another issue

Reply:
Fixed